starknet_py/tests/e2e/client/conftest.py

```diff
@@ -26,11 +26,13 @@
 directory = os.path.dirname(__file__)
 
 
-async def prepare_network(gateway_account_client: AccountClient) -> PreparedNetworkData:
+async def prepare_network(
+    new_gateway_account_client: AccountClient,
+) -> PreparedNetworkData:
     contract_compiled = Path(contracts_dir / "balance_compiled.json").read_text("utf-8")
 
     prepared_data = await prepare_net_for_tests(
-        gateway_account_client, compiled_contract=contract_compiled
+        new_gateway_account_client, compiled_contract=contract_compiled
```

Comment:
I wonder if this should be changed now, this way all transactions in our client tests are version 1 instead of 0.

Reply:
Yes, but we are fetching them only, so version doesn't matter here